src/pybind/mgr/orchestrator.py

```diff
@@ -1120,6 +1124,12 @@ def __repr__(self):
         return "<ServiceDescription>({n_name}:{s_type})".format(n_name=self.nodename,
                                                                   s_type=self.name())
 
+    def entity_name(self):
+        if self.service_type in ['osd','mon','mgr','mds']:
+            return self.name()
+        else:
+            return 'client.' + self.name()
```

Comment:
Does this make sense for **al**l other servcies? prometheus, grafana, alertmanager, rbd-mirror, rook-ceph-osd-prepare?

Reply:
nope.  i expect this will get switched around as soon as we add those services

Reply:
hm.